src/main/java/greencity/dto/openingHours/OpeningHoursDto.java

```diff
@@ -0,0 +1,34 @@
+package greencity.dto.openingHours;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import greencity.entity.OpeningHours;
+import greencity.entity.enums.WeekDay;
+import java.time.LocalTime;
+import javax.validation.constraints.NotNull;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+public class OpeningHoursDto {
+
+    @NotNull
+    @JsonFormat(pattern = "HH:mm", shape = JsonFormat.Shape.STRING)
+    @ApiModelProperty(dataType = "java.lang.String")
+    private LocalTime openTime;
+
+    @NotNull
+    @JsonFormat(pattern = "HH:mm", shape = JsonFormat.Shape.STRING)
+    @ApiModelProperty(dataType = "java.lang.String")
+    private LocalTime closeTime;
+
+    @NotNull private WeekDay weekDay;
+
+    public OpeningHoursDto(OpeningHours openingHours) {
```

Comment:
I think that you use modelmapper to map entity to dto and vice versa

Reply:
Yes , you are right , but it isn't my part of my code, I just resolved merge conflict in this class